docs_user/modules/openstack-glance_adoption.adoc

```diff
@@ -73,6 +73,95 @@ spec:
 '
 ----
 
+=== Using swift backend
+
+When Glance is deployed with Swift as a backend in the source environment based
+on TripleO, the podified `glanceAPI` instance should be deployed with the following
+configuration:
+
+----
+..
+spec
+  glance:
+   ...
+      customServiceConfig: |
+          [DEFAULT]
+          enabled_backends = default_backend:swift
+          [glance_store]
+          default_backend = default_backend
+          [default_backend]
+          swift_store_create_container_on_put = True
+          swift_store_auth_version = 3
+          swift_store_auth_address = {{ .KeystoneInternalURL }}
+          swift_store_endpoint_type = internalURL
+          swift_store_user = service:glance
+          swift_store_key = {{ .ServicePassword }}
+----
+
+It is recommended to write the patch manifest into a file, for example `glance_swift.patch`.
+For example, the Glance deployment with a Swift backend would look like this:
+
+----
+spec:
+  glance:
+    enabled: true
+    apiOverride:
+      route: {}
+    template:
+      databaseInstance: openstack
+      storageClass: "local-storage"
+      storageRequest: 10G
+      customServiceConfig: |
+          [DEFAULT]
+          enabled_backends = default_backend:swift
+          [glance_store]
+          default_backend = default_backend
+          [default_backend]
+          swift_store_create_container_on_put = True
+          swift_store_auth_version = 3
+          swift_store_auth_address = {{ .KeystoneInternalURL }}
+          swift_store_endpoint_type = internalURL
+          swift_store_user = service:glance
+          swift_store_key = {{ .ServicePassword }}
+      glanceAPIs:
+        default:
+          replicas: 1
+          override:
+            service:
+              internal:
+                metadata:
+                  annotations:
+                    metallb.universe.tf/address-pool: internalapi
+                    metallb.universe.tf/allow-shared-ip: internalapi
+                    metallb.universe.tf/loadBalancerIPs: 172.17.0.80
+              spec:
+                type: LoadBalancer
+          networkAttachments:
+          - storage
```

Comment:
I think this should be have additional 2 space indent to be inside of 'networkAttachments' section

Reply:
Makes sense, thanks: I already fixed it in the tests, but I didn't update the doc reference! Thank you, did it in my last update.